espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/SourceDocument.kt

```diff
@@ -199,9 +206,49 @@ class SourceDocument constructor(
         serializer?.endTag(NAMESPACE, tagName)
     }
 
+    private fun serializeComposeNode(semanticsNode: SemanticsNode?, depth: Int) {
+        if (semanticsNode == null) {
+            return
+        }
+        val nodeElement = ComposeNodeElement(semanticsNode)
+        serializer?.startTag(NAMESPACE, COMPOSE_TAG_NAME)
+
+        linkedMapOf(
+            ViewAttributesEnum.INDEX to { nodeElement.index },
```

Comment:
I would say it also makes sense to expose https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/SemanticsProperties#Password(), https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/SemanticsProperties#ProgressBarRangeInfo() and https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/SemanticsProperties#InvisibleToUser()